PIL/Image.py

```diff
@@ -586,14 +586,17 @@ def _dump(self, file=None, format=None):
     def __eq__(self, other):
         if self.__class__.__name__ != other.__class__.__name__:
             return False
-        a = (self.mode == other.mode)
-        b = (self.size == other.size)
-        c = (self.getpalette() == other.getpalette())
-        d = (self.info == other.info)
-        e = (self.category == other.category)
-        f = (self.readonly == other.readonly)
-        g = (self.tobytes() == other.tobytes())
-        return a and b and c and d and e and f and g
+
+        properties = ['mode', 'size', 'info', 'category', 'readonly']
+
+        props_eq = all(self.__getattribute__(p) == other.__getattribute__(p)
+                       for p in properties)
+        if not props_eq:
+            return False
+
+        funcs_eq = (self.getpalette() == other.getpalette()) \
+                   and (self.tobytes() == other.tobytes())
```

Comment:
Minor [pep8](https://travis-ci.org/python-pillow/Pillow/jobs/138750658#L4694) thing:
598: E127 continuation line over-indented for visual indent

Also I don't think it needs the `\` because it's all inside brackets.